src/main/java/org/folio/rest/impl/InstanceBatchSyncAPI.java

```diff
@@ -36,8 +38,12 @@ public void postInstanceStorageBatchSynchronous(boolean upsert, InstancesPost en
     final List<Future> futures = new ArrayList<>();
     final HridManager hridManager = new HridManager(Vertx.currentContext(), postgresClient);
 
+    SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+    final String currentDate = sdf.format(new Date());
```

Comment:
Given this code appears in this class, `InstanceStorageAPI` and `InstanceStorageBatchAPI`, could it be worth extracting a shared method to do this? It might also help describe the intent of this code.

Reply:
Can you explain further what you mean by shared method?  Do you mean pulling the code out and putting it in a utility class they all use?  I could certainly do that.

Reply:
> Do you mean pulling the code out and putting it in a utility class they all use?

Yes